faf-commons-api/src/main/java/com/faforever/commons/api/dto/MapPoolAssignment.java

```diff
@@ -0,0 +1,23 @@
+package com.faforever.commons.api.dto;
+
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class MapPoolAssignment {
```

Comment:
Needs to `implements ElideEntity` and @EqualsAndHashCode for id.
Even better: let's enrich it once more so it can `extends AbstractEntity`

Reply:
Aight